site/content/en/docs/tutorials/volume_snapshots_and_csi.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "CSI Driver and Volume Snapshots"
+linkTitle: "CSI Driver and Volume Snapshots"
+weight: 1
+date: 2020-08-06
+description: >
+  CSI Driver and Volume Snapshots
+---
+
+## Overview
+
+This tutorial explains how to set up the CSI Hostpath Driver in Minikube and create volume snapshots.
+
+## Prerequisites
+
+- Minikube 1.12.3 with Kubernetes 1.18.x or higher
```

Comment:
I would just say latest version of minikube here, since 1.12.3 has already been released